torch/optim/lr_scheduler.py

```diff
@@ -8,11 +8,12 @@
 
 
 class _LRScheduler(object):
-    def __init__(self, optimizer, last_epoch=-1):
+    def __init__(self, optimizer, last_epoch=-1, min_lr=0.0):
         if not isinstance(optimizer, Optimizer):
             raise TypeError('{} is not an Optimizer'.format(
                 type(optimizer).__name__))
         self.optimizer = optimizer
+        self.min_lr = min_lr
```

Comment:
This needs to have `__setstate__` implemented. Serialized instances won't have this attribute